charts/akeyless-api-gateway/values.yaml

```diff
@@ -18,9 +18,12 @@ initContainer:
     pullPolicy: IfNotPresent
 
 cache:
+  tls:
+    autoGenerated: false
+    existingSecretName: ""
```

Comment:
duplicate with existingCacheConnectionSecret??

Reply:
one is for tls the other is the redis password